model/schema/phenotypeAndDiseaseAnnotation.yaml

```diff
@@ -121,6 +116,18 @@ classes:
       - with
       - evidence_code           # added to avoid the need for a publication join
       - date_assigned           # dateAssigned in JSON schema
+      - inferred_gene
+      - inferred_gene_association
```

Comment:
@chris-grove just wondering what the type is for this `inferred_gene_association` I would guess that if we have the association we might not need the inferred_gene as it would come with the association. Same with Allele?